ontology/stato.md

```diff
@@ -18,10 +18,34 @@ license:
 preferredPrefix: STATO
 products:
 - id: stato.owl
+publications:
+- id: https://doi.org/10.1038/s41597-019-0286-0
+  title: Experiment design driven FAIRification of omics data matrices, an exemplar
+- id: https://doi.org/10.1038/s41597-020-0409-7
+  title: Semantic concept schema of the linear mixed model of experimental observations
 repository: https://github.com/ISA-tools/stato
 tags:
 - statistics
 tracker: https://github.com/ISA-tools/stato/issues
+usages:
+- description: struct (Statistics in R using Class-based Templates), Struct integrates with the STATistics Ontology to ensure consistent reporting and maximizes semantic interoperability
+  publications:
+  - id: https://doi.org/10.1101/2021.10.10.463703
```

Comment:
There is a mismatch between the publication DOI and the title. The DOI refers to a paper that has the title 'Brain Data Standards - A method for building data-driven cell-type ontologies' where as the 'title' field is referring to 'struct: an R/Bioconductor-based framework for standardized metabolomics data analysis and beyond'.

Did you mean to use https://pubmed.ncbi.nlm.nih.gov/33325493 instead?

Reply:
indeed !